### PR TITLE
Some wording improvements from Abigail Phillips in Legal.

### PR DESCRIPTION
**Pull request by gerv**
None of these changes are changes to policy.

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/mpl/license-policy.html [hunk 1]
```diff
@@ -15,14 +15,15 @@
...
+  <h1>Mozilla Source Code License Policy</h1>
   <p class="version"><em>Version 4.0</em></p>
 
   <section>
     <h2 id="Introduction">Introduction</h2>
     <p><em>Overview: <a href='{{ static("img/mozorg/mpl/license-policy-flowchart.png") }}'>this flowchart</a> provides a summary of the policy that can help you decide which license to use.</em></p>
-    <p>This policy determines which license to use for source files stored in Mozilla-controlled code repositories, whether hosted at Mozilla or elsewhere - "Mozilla Repositories". Committers to Mozilla Repositories must comply with this policy, as they agreed to do in the <a href="http://mozilla.org/hacking/committer/committers-agreement.pdf">Committer's Agreement</a>.</p>
-    <p>The following is a brief summary of the Mozilla license policy:</p>
+    <p>This policy determines which license to use for source files stored in Mozilla-controlled code repositories, whether hosted at Mozilla or elsewhere - "Mozilla Repositories". Committers to Mozilla Repositories must comply with this policy, as they agreed to do in the <a href="http://mozilla.org/hacking/committer/committers-agreement.pdf">Committer's Agreement</a>.
```
Comment (pmclanahan) [inline bedrock/mozorg/templates/mozorg/mpl/license-policy.html]: Be good to go ahead and update this link to the new one and avoid some redirects for the user:

https://static.mozilla.com/foundation/documents/commit-access/committers-agreement.pdf

**Comment by gerv** [comment]
URL updated as requested.

**Comment by pmclanahan** [comment]
Looks fine to me then. @craigcook any objections?

**Comment by craigcook** [comment]
Looks good to me. Is there a bug number we should reference with the commit?

**Comment by gerv** [comment]
Not at the moment; she just emailed me a redline and I took some of her suggestions. I can open one if you like. But no policy change is implied.

**Comment by craigcook** [comment]
No bug required. I've squashed the two commits and merged in https://github.com/mozilla/bedrock/commit/5df5bde5c5199f20a8f7ce0238d21ccd50cecb73
Thanks!

**Pull request closed without merging**